src/Symfony/Component/DependencyInjection/Compiler/AliasDeprecatedPublicServicesPass.php

```diff
@@ -17,18 +17,7 @@
 
 final class AliasDeprecatedPublicServicesPass extends AbstractRecursivePass
 {
-    private $tagName;
-
-    private $aliases = [];
-
-    public function __construct(string $tagName = 'container.private')
-    {
-        if (0 < \func_num_args()) {
-            trigger_deprecation('symfony/dependency-injection', '5.3', 'Configuring "%s" is deprecated.', __CLASS__);
```

Comment:
Removing deprecated features should be done on its own rather than in a PR saying "add types" IMO

Reply:
Good point, I'll do that.

Reply:
#41932